### DISCUSSION TITLE
windows 11 style new notification icon in system tray

### DISCUSSION THREAD

**Discussion by chip33** [valinet/ExplorerPatcher#752] [Ideas]
I recently tried windows 11 taskbar for debugging purposes.
One useful new feature was the small icon by the clock that appears if you have new notifications.
![Screenshot 2022-01-28 171800](https://user-images.githubusercontent.com/48499740/151611034-5eea4ef2-b028-4df8-81d9-235e4a88aa47.png)

It could be good if that could be ported or emulated, maybe by adding it as an optional system tray icon that only exists when new notifications arrive.
Users could then drop it down next to the clock and it would function in the same way opening notifications panel on click.

**Comment by valinet** [comment]
Or, just use the standard notification button that already does this. What’s wrong with that?

**Reply by chip33** [reply]
Ah yes, I see, it is because I have enabled the network icon as opening control centre with the tweak you added before to replace action centre. That is why I get no notice of notifications.
In this case the new button is useful.

**Reply by valinet** [reply]
I have checked it just now, it shows the number of unread notifications, functionally similar to what the new button offers.

![image](https://user-images.githubusercontent.com/6503598/151615958-382e731a-7db1-411d-b076-df61bad15d04.png)

**Reply by chip33** [reply]
hi, yes, i edited my reply above to say why I was not seeing them.
Looks like when you turn off the action centre button it automatically enables this small notification circle in windows 11 as an alternative. Which is neat.

**Reply by valinet** [reply]
The circle is there at all times (when there are notifications). That’s the notification count indicator on the new taskbar.

**Reply by chip33** [reply]
In Windows 11 the calendar and notification flyouts are combined in "Win+N Notification Center" tied to the clock so the notification count indicator is needed and works well at simplifying the system tray as it only appears when needed.
In Windows 10 taskbar with EP the clock and the action center now both include the calendar so it often makes sense to set the clock to Windows 11 flyout and turn off action center to avoid duplication.
This leaves the functionality provided by the notification count indicator missing.

**Reply by valinet** [reply]
Yeah, I understand, I do the same, the difference is, I don’t care about the notifications 😂, the important ones I also get on my phone. Anyway, I will look into it.

**Reply by Flare-San** [reply]
Very helpful discussion, thanks chip33 and valinet.

So using the tool-settings:
Task bar: choosing taskbar-style:  "Windows 10"
then klicking "Customize system icons in the notification area" and disable Info-Center icon there
then
System tray: Clock: "Winows 11 flyout"
then lower left corner press "Restart File Explore"
got me there

but now there is no icon at all that could show me a notification-counter
i would like a auto-hiding Info-Center icon when ther are zero new notifications

thanks for your great work valinet, much appreciation
kind regards, Flare-San

**Reply by chip33** [reply]
One way to do this would be to make the Action Center icon open Control Center when clicked.
Then you would not need the Network icon. 
Just the Clock showing Win 11 flyout and the icon of the Action Center changing to show notification status.

**Reply by chip33** [reply]
I was wondering if it is a lot harder to make Action Center icon open Control Center than opening it with the Network icon.
Maybe I have not thought or a possible reason this is not practical but it could be a nice solution to showing notifications with just the Clock and Action Center icon needed in the system tray for full functionality.

**Comment by TheGamer1445891** [comment]
@Amrsatrio please add this 😭🙏

**Comment by Cyber5imon** [comment]
This would be fantastic!
